src/components/Watch/FullScreen/FullScreen.tsx

```diff
@@ -0,0 +1,52 @@
+import "./FullScreen.css";
+import FullscreenIcon from "@mui/icons-material/Fullscreen";
+import FullscreenExitIcon from "@mui/icons-material/FullscreenExit";
+
+interface FullScreenProps {
+  videoRef: React.RefObject<HTMLVideoElement>;
+  isFullScreen: boolean;
+}
```

Comment:
```suggestion
type VideoElementWithFullScreen = HTMLVideoElement & {
  mozRequestFullScreen?: () => void;
  webkitRequestFullscreen?: () => void;
  msRequestFullscreen?: () => void;
}

interface FullScreenProps {
  videoRef: React.RefObject<VideoElementWithFullScreen>;
  isFullScreen: boolean;
}
```

requestFullScreen에 브라우저별 prefix가 붙은 메소드들이 타입스크립트의 내장 타입인 HMTLVideoElement에는 준비되어있지 않은 것으로 보입니다. 만약 브라우저별 prefix가 붙은 메소드를 꼭 사용해야 한다면 위처럼 HTMLVideoElement에 필요한 메소드들이 optional하게 붙은 type alias를 새로 만들어 사용해주는 게 좋은 것 같습니다.

아래에서 나올 Document도 마찬가지일 듯 합니다